package.json

```diff
@@ -39,6 +39,6 @@
         "mixwith": "^0.1.1",
         "underscore": "^1.8.3",
         "underscore.string": "^3.3.4",
-        "@exabyte-io/periodic-table.js": "2.0.4"
+        "@exabyte-io/periodic-table.js": "file:/stack/lib/periodic-table"
```

Comment:
To run this code you need to link to periodic-table branch SOF-5214.

Reply:
Then maybe we should point to that branch/hash instead inside when committing the code??